core/pbcc_entity_api.c

```diff
@@ -0,0 +1,687 @@
+/* -*- c-file-style:"stroustrup"; indent-tabs-mode: nil -*- */
```

Comment:
This looks specific to Objects API changes.

1. Should we include this right now or move it to later?
2. Should the file be `pbcc_objects_api.c` ?

Reply:
It was some work to remove it, so, I opted out to leave it and just update it as we finish up.

Reply:
That's fine then, thanks for the explanation.